ReSharper.FSharp/src/FSharp.Psi.Features/FSharp.Psi.Features.fsproj

```diff
@@ -200,7 +200,7 @@
     <Compile Include="src\QuickFixes\ReplaceWithAbbreviatedTypeFix.fs" />
     <Compile Include="src\QuickFixes\UseWildSelfIdFix.fs" />
     <Compile Include="src\QuickFixes\SpecifyParameterTypeFix.fs" />
-    <Compile Include="src\QuickFixes\ReplaceWithInequalityOperatorFix.fs" />
+    <Compile Include="src\QuickFixes\ReplaceWithFSharpOperatorFix.fs" />
```

Comment:
I wonder where `FSharp` comes from? `==` and `!=` are totally valid F# operators. Just are not predefined/built-in.